app/controllers/providers/allocations_controller.rb

```diff
@@ -49,6 +49,17 @@ def show
       end
     end
 
+    def new_request
```

Comment:
Currently the 'new' action is for 're-requests' so we may need to have a think about actions / naming across the board for both requests and re-requests

Reply:
We had a chat and for urls we decided:
- `re-request` for the 'request again' flow
- `request` for the new request (since we can't use `request` as a name in the controller) I went for `new_request`. 
Do you think `new` would be better here? If so, we should also agree the new names for the `re-request` flow.

Reply:
Hey @AgaDufrat - have commented here ... https://github.com/DFE-Digital/publish-teacher-training/pull/1045/files#r416136962